iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -446,29 +439,105 @@ TEST_F(PortManager_test, ConditionVariablesDestroy)
     }
 }
 
+TEST_F(PortManager_test, NodeDataOverflow)
+{
+    std::string process = "Process";
+    std::string node = "Node";
+
+    // first acquire all possible NodeData
+    for (unsigned int i = 0U; i < iox::MAX_NODE_NUMBER; i++)
```

Comment:
I would split up this test into multiple ones.
- AcquiringMaximumNumberOfNodesIsWorking
- AcquiringOneMoreThenMaximumNumberOfNodesFails
- DeletingNodeWorks (fill up to maximum, delete one and add one)